.eslintrc.js

```diff
@@ -1,10 +1,110 @@
 module.exports = {
+	root: true,
+	globals: {
+		__DEV__: true,
+		process: true,
+		window: true,
+		document: true
+	},
 	env: {
 		node: true
 	},
-	extends: ['enact', 'plugin:prettier/recommended', 'prettier'],
-	plugins: ['import'],
+	extends: ['eslint:recommended', 'plugin:prettier/recommended', 'prettier'],
+	parser: '@babel/eslint-parser',
+	parserOptions: {
+		ecmaFeatures: {
+			jsx: true
+		},
```

Comment:
Do we need this?

Reply:
This option is for `eslint-plugin-react` 7.32.x: https://github.com/enactjs/cli/pull/312/files#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5R8

So we don't need this because we will not linting react code.
I will remove it.